.gitignore

```diff
@@ -13,12 +13,7 @@ version*
 
 # IDE files
 .idea/
-
-# Test-generated files
```

Comment:
The tests are meanwhile running in `docker` containers and create the files where a live system does this as well (`/etc/pihole`, `/var/log/`).